src/shim/ShimUtils.ts

```diff
@@ -88,6 +88,16 @@ export function dedupeArray(list: string[], clearEmptyValues = false) {
     });
 }
 
+/**
+ * Validates that either an owner was specified or addAsAdmin is true allowing the creator to be the default owner.
+ * An error will be returned if no owner is specified and the group creator is not going to be an admin.
+ */
+export function validateOwnership(addAsAdmin?: boolean, maybeOwner?: string) {
```

Comment:
Pretty minor, but we should add a unit test for this method.

Reply:
done